src/main/java/com/karankumar/bookproject/backend/service/PredefinedShelfService.java

```diff
@@ -36,9 +51,12 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+
+
+
 /**
- * A Spring service that acts as the gateway to the @see PredefinedShelfRepository -- to use the
- * PredefinedShelfRepository, you should go via this PredefinedShelfService.
+ * A Spring service that acts as the gateway to the {@code ShelfRepository} -- to use the {@code ShelfRepository}, a
```

Comment:
Could you undo this change? `ShelfRepository` does not exist, but `PredefinedShelfRepository` does